src/dlgprefreplaygaindlg.ui

```diff
@@ -258,7 +264,7 @@
       <item row="0" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
-         <string>The normal average ReplayGain playback level is -14 dB. You may increase it using the inital boost, if your tracks does not require the full dynamic headroom, or reduce it if you experience that your tracks are clipping by default.</string>
+         <string>The loudness of the ReplayGain reference signal is -18 LUFS (Loudness Units relative to Full Scale). You may increase it, if your tracks does not require the full dynamic headroom or reduce it if you experience that your tracks are clipping by default. The standard loudness for European TV is -23 LUFS and for podcasting it is -16 LUFS.</string>
```

Comment:
add a link

"ReplayGain targets a reference loudness of -18 LUFS (Loudness Units relative to Full Scale). You may increase it if you find Mixxx is too quiet or reduce it if you find that your tracks are clipping.  Learn more about LUFS and loudness here: [link]"

We should not mention European TV, that's a culturally-specific reference that will not make sense to people outside of Europe.  Also, Mixxx is not designed to broadcast to TV.  I could see mentioning the podcasting level, or maybe if icecast has a target loudness we could mention that because we have so many streaming radio users.  but I really think that should go in a link.